<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crossection</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>Crossection</string>
    </attribute>
    <widget class="QWidget" name="tab" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>401</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>General notes :</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>421</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Select model :</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PyMol Objects :</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>231</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Profile options</string>
       </property>
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Slice Spacing :</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>151</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Profile depth</string>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>80</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Full profile</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Set profile depth</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Depth :</string>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Select all atoms for surface</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Store crossection images</string>
       </property>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>181</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Keep raw slice images</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Import object list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Make surface</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Add hydrogens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Make profile</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
